tutorindigo/templates/indigo/env.config.jsx

```diff
@@ -1,8 +1,39 @@
-import React from 'react';
+import React, { useEffect } from 'react';
+import Cookies from 'universal-cookie';
 
 import Footer from '@edly-io/indigo-frontend-component-footer';
+import { getConfig } from '@edx/frontend-platform';
 import { DIRECT_PLUGIN, PLUGIN_OPERATIONS } from '@openedx/frontend-plugin-framework';
 
+let themeCookie = 'indigo-toggle-dark';
+let themeCookieExpiry = 90; // days
+
+const AddDarkTheme = () => {
+  const cookies = new Cookies();
+  const isThemeToggleEnabled = getConfig().INDIGO_ENABLE_DARK_TOGGLE;
```

Comment:
I am unable to see the toggle in the MFEs, so I'm guessing this is not working as expected. The LMS config API call is properly returning `INDIGO_ENABLE_DARK_TOGGLE: true`. Can you please troubleshoot?

Reply:
Yes, header package is not published yet in which we are using `INDIGO_ENABLE_DARK_TOGGLE `. Here's the [PR](https://github.com/edly-io/frontend-component-header/pull/21) for header.  I am currently testing the header package and then publish it. 
Then, I have to update header package version here too.

Reply:
Publishing the header is done and the version is updated too. I've tested on my system and openedx and mfe image building is successful and working. I will test on sandboxdev and inform the results here.

Reply:
I tested on sandboxdev and it's working fine. Steps I've taken:
```bash
tutor local stop
rm -rf tutor/env
pip install git+https://.......this branch
tutor config save
tutor images build openedx
tutor images build mfe
tutor local start -d


tutor  config save  --set INDIGO_ENABLE_DARK_TOGGLE=false
tutor images build openedx
tutor local start -d


tutor config save --set INDIGO_ENABLE_DARK_TOGGLE=true
tutor images build openedx
tutor local start -d
```